src/renderer/deviceMessages.ts

```diff
@@ -18,4 +18,14 @@ export function updateDeviceChats(accountId: number) {
 
 Full Changelog: https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md#1_34_0`
   )
+
+  BackendRemote.rpc.addDeviceMessage(
+    accountId,
+    'changelog-version-1.34.1-version0',
+    `What's new in 1.34.1?
+
+✨ make some webxdcs work again
```

Comment:
```suggestion
✨ Make some Private Apps work again
```

"webxdc" are "Private Apps" in UI.

i think, the 1.34.1 device message could also be omitted (and say "What's new in 1.34" above). 1.34.0  was not widely released and for bugfix releases there are not always device messages afaik.

Reply:
tried not to scare apple, but sure probably makes sense to omit the message